src/test/java/com/feelmycode/parabole/controller/CouponControllerTest.java

```diff
@@ -0,0 +1,363 @@
+                            .description("첫 페이지 여부"),
+                        fieldWithPath("data.numberOfElements").type(JsonFieldType.NUMBER)
+                            .description("총 항목 갯수"),
+                        fieldWithPath("data.empty").type(JsonFieldType.BOOLEAN)
+                            .description("빈 여부"),
+                        fieldWithPath("data.last").type(JsonFieldType.BOOLEAN)
+                            .description("마지막 페이지 여부"),
+                        fieldWithPath("data.size").type(JsonFieldType.NUMBER).description("페이지당 항목 수"),
+                        fieldWithPath("data.number").type(JsonFieldType.NUMBER).description("현재 페이지"),
+                        fieldWithPath("data.sort").type(JsonFieldType.OBJECT).description("정렬 정보"),
+                        fieldWithPath("data.sort.empty").type(JsonFieldType.BOOLEAN)
+                            .description("정렬 정보 여부")
+                    )
+                )
+            )
+            .when()
+            .port(port)
+            .get("/api/v1/coupon/list");
+
+        // Then
+        Assertions.assertEquals(HttpStatus.OK.value(), resp.statusCode());
+
+    }
+
+
+    @Test
+    @DisplayName("셀러 본인이 발행한 쿠폰 목록 조회 : /coupon/list 에 셀러 계정으로 테스트")
+    public void couponListSeller() {
+
+        Long userId = 37L;                                                                          // sellerId 3
+        String userToken = this.tokenProvider.create(
+            User.builder().id(userId).email("r").password("r").nickname("r").build());              // 유저 계정
+
+        // When
+        Response resp = given(this.spec)
+            .header("authorization", "Bearer " + userToken)
+            .accept(ContentType.JSON)
+            .contentType(ContentType.JSON)
+            .filter(document("coupon-list-seller",
+                    preprocessRequest(modifyUris()
+                            .scheme("https")
+                            .host("parabole.com"),
+                        prettyPrint()),
+                    preprocessResponse(prettyPrint()),
+                    responseFields(
+                        fieldWithPath("success").type(JsonFieldType.BOOLEAN).description("성공여부"),
+                        fieldWithPath("message").type(JsonFieldType.STRING).description("메세지"),
+                        fieldWithPath("data").type(JsonFieldType.OBJECT).description("응답 정보"),
+                        fieldWithPath("data.content").type(JsonFieldType.ARRAY).description("쿠폰 목록"),
+                        fieldWithPath("data.content.[].couponId").type(JsonFieldType.NUMBER)
+                            .description("쿠폰 ID").optional(),
+                        fieldWithPath("data.content.[].name").type(JsonFieldType.STRING)
+                            .description("쿠폰 이름").optional(),
+                        fieldWithPath("data.content.[].type").type(JsonFieldType.STRING)
+                            .description("쿠폰 유형").optional(),
```

Comment:
쿠폰 유형에 어떤것이 있는지 설명에 작성해주세요

아래는 예시이고 알맞게 수정해주세요

```suggestion
                        fieldWithPath("data.content.[].type").type(JsonFieldType.STRING)
                            .description("쿠폰 유형 (정률쿠폰: RATE / 정액쿠폰: AMOUNT)").optional(),
```